### ISSUE TITLE
Cursor disappears

### ISSUE THREAD

**Issue by venkat-gvk** [OpenBoard-org/OpenBoard#477]
Hello open board team, 
        I don't know this is the correct title for the question. But, Is there any way to make the cursor visible while drawing or writing as the whiteboard or OneNote does? Once I start writing the cross disappears and comes back when I stop. Is there a way to make it persist while drawing or writing?

**Comment by letsfindaway** [comment]
No, currently not. This behavior is fixed in OpenBoard and not configurable.

**Comment by kaamui** [comment]
Could be not that hard to make it configurable though. But due to our limited resources and our current priorities, I won't be able to work on it soon.

**Comment by RickGxG** [comment]
Maybe this answer can help for those using OpenBoard and Wacom tablet in Ubuntu 22.04. Posted first: https://github.com/OpenBoard-org/OpenBoard/issues/514

I had a similar problem in Ubuntu 22.04 with OpenBoard 1.6.4 and a basic Wacom One tablet. It seems to be a problem with Qt and Wayland. I think the problem arises because OpenBoard is always in fullscreen. I have three workarounds:

1. Activate "Locate Pointer" in Ubuntu Settings > Accessibility. When the Ctrl key is pressed, the tablet pointer becomes visible for a moment. 
2. Select the "MultiScreen" option from the OpenBoard Menu (the little gear icon located below the close button). This action breaks the fullscreen mode and enables the tablet pointer visualization. Note: Some actions, like opening saved slides, might set OpenBoard back to fullscreen mode. 
3. Uninstall the old version and install OpenBoard 1.7.3 (in my case for Ubuntu 22.04, from [here](https://github.com/OpenBoard-org/OpenBoard/wiki/Downloads#other-platforms)). This version has the RunInWindow parameter, which must be set to true in OpenBoardUser.config file, normally located in the .local/share/OpenBoard/ folder. See OpenBoard [Configuration](https://github.com/OpenBoard-org/OpenBoard/wiki/Configuration) and [Parameters](https://github.com/OpenBoard-org/OpenBoard/wiki/Parameters). 

I think the last procedure is the best, even though it doesn't fix the root cause.
